### ISSUE TITLE
Decreased CNS synapse formation

### ISSUE THREAD

**Issue by sarahstofel** [obophenotype/human-phenotype-ontology#7066]
**Preferred term label:**

Decreased CNS synapse formation

**Synonyms**

reduced CNS synapse formation

**Definition (free text, please give PubMed ID)**

A reduction in the frequency of the process of generating the initial connections between an axon and effector tissue or neuron.

Please refer to MP:0004998

**Parent term (use hpo.jax.org/app)**

Morphological central nervous system abnormality 

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

OMIM: 104300

**Your nano-attribution (ORCID)**

https://orcid.org/0000-0003-2307-1226

**Comment by pnrobinson** [comment]
@sarahstofel  The rate of synapse formation is not something that is measured in human medicine afaik. Would a term representing synaptic density be appropriate? See for instance this article https://stm.sciencemag.org/content/8/348/348ra96

**Comment by pnrobinson** [comment]
I think for now this is not in scope for the HPO, this is not something that is measured in clinical practice.
